src/components/networks/createNetworkDialog.jsx

```diff
@@ -420,41 +424,16 @@ class CreateNetworkModal extends React.Component {
     }
 }
 CreateNetworkModal.propTypes = {
-    close: PropTypes.func.isRequired,
-    devices: PropTypes.array.isRequired,
 };
```

Comment:
This empty proptypes object doesn't make much sense, I guess it was left here by accident, and should be deleted:
```
CreateNetworkModal.propTypes = {
};
```

Reply:
Done.